src/main/proto/sqlflow.proto

```diff
@@ -63,12 +63,16 @@ message FetchRequest {
 }
 
 message FetchResponse {
-    message Logs {
+    message Logs {  //  depreated
         repeated string content = 1;
     }
+    message Responses {
+        repeated Response response = 1;
+    }
     FetchRequest updated_fetch_since = 1;
     bool eof = 2;
-    Logs logs = 3;
+    Logs logs = 3;  //  depreated
```

Comment:
Do we need to update the code of show logs, in the newly protobuf,
the log is in `Response.Message()`

Reply:
Sure.